src/main/java/org/killbill/billing/plugin/vertex/VertexTaxCalculator.java

```diff
@@ -285,6 +287,16 @@ private Collection<InvoiceItem> toInvoiceItems(final UUID invoiceId,
         }
     }
 
+    private static String getDescription(final TaxesType transactionLineDetailModel) {
+        String description = MoreObjects.firstNonNull(transactionLineDetailModel.getTaxCode(), MoreObjects.firstNonNull(transactionLineDetailModel.getVertexTaxCode(), "Tax"));
```

Comment:
I think this earlier construction of description is not needed as in general case it will be overwritten after getting jurisdiction, why don't we init description when we know jurisdiction?

Reply:
jurisdiction always will have value?

Reply:
per schema it is nullable, but it is unlikely that it will be missing in real scenarios I guess. 

- We do smth everytime and we know that sometimes it is not needed and it easily can be written using if/else branch to don't have unneeded operation. 
- Also imo for code reader it is additional complexity to track description variable.

Reply:
changed